src/index.js

```diff
@@ -1 +1,29 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { getScores, addScore } from './modules/fetchLeaderboard.js';
+import renderLeaderboard from './modules/renderLeaderboard.js';
+
+const reloadBtn = document.getElementById('refresh-btn');
+const form = document.getElementById('add-form');
+const nameToAdd = document.getElementById('name');
+const scoreToAdd = document.getElementById('score');
+
+const renderListOnload = async () => {
+  const newList = await getScores();
+  renderLeaderboard(newList);
+};
+
+window.onload = renderListOnload();
+
+reloadBtn.addEventListener('click', async () => {
+  const newList = await getScores();
+  renderLeaderboard(newList);
+});
+
+form.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  await addScore(nameToAdd.value, scoreToAdd.valueAsNumber);
+  const newList = await getScores();
+  renderLeaderboard(newList);
```

Comment:
- [OPTIONAL] These two lines are not needed so you can remove them, because the requirement for submit button is to only hit the GET API request, and upon submitting the score should not reflect on the leaderboard. it should reflect when the user hit the refresh button.

Reply:
Perfect! I misunderstood that part. Thanks for your review! @Zuhaib042